code/game/machinery/recharger.dm

```diff
@@ -162,10 +162,13 @@
 	..()
 
 
-/obj/machinery/recharger/update_icon(using_power = 0)	//we have an update_icon() in addition to the stuff in process to make it feel a tiny bit snappier.
+/obj/machinery/recharger/update_icon(using_power = 0, scan = 0)	//we have an update_icon() in addition to the stuff in process to make it feel a tiny bit snappier.

```

Comment:
You can remove the `= 0` here, byond initialize variables as null, and in this case it doesn't matter between 0/null.